src/main/java/useCases/Login/LoginInputBoundary.java

```diff
@@ -1,8 +1,16 @@
 package useCases.Login;
 
-import useCases.CreateAccount.CreateAccountRequestModel;
-
+/**
+ * An interface which hides the details of how a User is logged into the application
+ */
 public interface LoginInputBoundary {
-    LoginResponseModel create(CreateAccountRequestModel requestModel);
+
+    /**
+     * The method which logs in a User
+     *
```

Comment:
Good documentation. I see that most, if not all, classes and methods are documented.